packages/@uppy/dashboard/types/index.d.ts

```diff
@@ -88,11 +88,13 @@ export default Dashboard
 
 export type DashboardFileEditStartCallback<TMeta extends IndexedObject<any>> = (file?: UppyFile<TMeta>) => void;
 export type DashboardFileEditCompleteCallback<TMeta extends IndexedObject<any>> = (file?: UppyFile<TMeta>) => void;
+export type DashboardShowPlanelCallback = (id: UIPlugin<DashboardOptions>['id']) => void;
```

Comment:
ID can be any string so I don't think we need to do something fancy here.

```suggestion
export type DashboardShowPlanelCallback = (id: string) => void;
```